tests/cli/test_import_schema.py

```diff
@@ -0,0 +1,490 @@
+import os
+
+import pytest
+
+from strawberry.cli.commands.schema_importer import sdl_importer
+
+
+@pytest.fixture
+def schema():
+    path = os.path.join(os.getcwd(), "tests", "cli", "helpers", "swapi_schema.gql")
+    with open(path, "r") as f:
+        return f.read()
+
+
+# Complex object
+def test_import_whole_schema(schema):
+    output = sdl_importer.import_sdl(schema)
+    assert output
```

Comment:
let's remove this test 😊 I think it is fine to test small, like done below :)

Reply:
Sure no problem!